cupy/lib/polynomial.pyx

```diff
@@ -4,6 +4,16 @@ import cupy
 from cupy.core.core cimport ndarray
 
 
+def _fft_poly(seq):
+    n = seq.shape[0]
+    if n == 1:
+        return seq[0]
+    if n == 2:
+        return cupy.math.misc._fft_convolve(seq[0], seq[1], 'full')
+    return cupy.math.misc._fft_convolve(_fft_poly(seq[:n/2]),
+                                        _fft_poly(seq[n/2:]), 'full')
```

Comment:
This implementation calls FFT O(n) times, but we can achieve O(log n) times if we support Nd convolve.

Reply:
You suggest supporting scipy.signal.convolve or extending the functionality of cupy.convolve ?

Reply:
OK!